agr_indexer/src/main/java/org/alliancegenome/indexer/rest/interfaces/EcoTermRESTInterface.java

```diff
@@ -0,0 +1,19 @@
+package org.alliancegenome.indexer.rest.interfaces;
+
+import com.fasterxml.jackson.annotation.JsonView;
+import jakarta.ws.rs.*;
+import org.alliancegenome.curation_api.model.entities.ontology.ECOTerm;
+import org.alliancegenome.curation_api.response.ObjectResponse;
+import org.alliancegenome.curation_api.view.View;
+
+@Path("/ecoterm")
+@Produces({"application/json"})
+@Consumes({"application/json"})
+public interface EcoTermRESTInterface {
+
+	@GET
+	@Path("/{curie}")
+	@JsonView({View.FieldsAndLists.class})
+	ObjectResponse<ECOTerm> find(@PathParam("curie") String curie);
```

Comment:
Does the interface in the curation api not work for this?

Reply:
I have not tried but we always had trouble with it.

Reply:
Due to Json view issues but that shouldn't be a problem here.

Reply:
We wanted a different view so we had to reimplement the interfaces. However in the end we made the change on the Curation side. So I would guess we don't need the Annotation Serivces / Interfaces anymore either.